test/setupFile.js

```diff
@@ -3,6 +3,7 @@ import 'babel-polyfill';
 jest.mock('~/components/shell/shell');
 
 global.fetch = require('jest-fetch-mock');
+global.regeneratorRuntime = regeneratorRuntime;
```

Comment:
That's an outdated change, but there is a different fetch package that could be swapped out.